wrapper/stmt.go

```diff
@@ -243,3 +243,283 @@ func TaosStmtUseResult(stmt unsafe.Pointer) unsafe.Pointer {
 func TaosStmtClose(stmt unsafe.Pointer) int {
 	return int(C.taos_stmt_close(stmt))
 }
+
+//TaosStmtSetSubTBName int        taos_stmt_set_sub_tbname(TAOS_STMT* stmt, const char* name);
+func TaosStmtSetSubTBName(stmt unsafe.Pointer, name string) int {
+	cStr := C.CString(name)
+	defer C.free(unsafe.Pointer(cStr))
+	return int(C.taos_stmt_set_sub_tbname(stmt, cStr))
+}
+
+//typedef struct TAOS_MULTI_BIND {
+//int            buffer_type;
+//void          *buffer;
+//uintptr_t      buffer_length;
+//int32_t       *length;
+//char          *is_null;
+//int            num;
+//} TAOS_MULTI_BIND;
+
+// TaosStmtBindParamBatch int        taos_stmt_bind_param_batch(TAOS_STMT* stmt, TAOS_MULTI_BIND* bind);
+func TaosStmtBindParamBatch(stmt unsafe.Pointer, multiBind [][]interface{}, bindType []*taosTypes.ColumnType) int {
+	var binds = make([]C.TAOS_MULTI_BIND, len(multiBind))
+	var needFreePointer []unsafe.Pointer
+	defer func() {
+		for _, pointer := range needFreePointer {
+			C.free(pointer)
+		}
+	}()
+	for columnIndex, columnData := range multiBind {
+		bind := C.TAOS_MULTI_BIND{}
+		//malloc
+		rowLen := len(multiBind[0])
+		bind.num = C.int(rowLen)
+		nullList := unsafe.Pointer(C.malloc(C.ulong(rowLen)))
+		needFreePointer = append(needFreePointer, nullList)
+		lengthList := unsafe.Pointer(C.malloc(C.ulong(rowLen * 4)))
+		needFreePointer = append(needFreePointer, lengthList)
+		var p unsafe.Pointer
+		columnType := bindType[columnIndex]
+		switch columnType.Type {
+		case taosTypes.TaosBoolType:
+			//1
+			p = unsafe.Pointer(C.malloc(C.ulong(rowLen)))
+			bind.buffer_type = C.TSDB_DATA_TYPE_BOOL
+			bind.buffer_length = C.uintptr_t(1)
+			for i, rowData := range columnData {
+				currentNull := unsafe.Pointer(uintptr(nullList) + uintptr(i))
```

Comment:
> Just keep in mind:
> 
> primitive data types (bool/integers/floats) has NULL values, eg. BOOL_NULL = 0x02, TINYINT_NULL = -128.
> 
> So that you could skip the null list allocation.

if the null list can be skipped, then this may prove the design of the interface is defective.
in addition, if I use the special null value, I think taosdef.h should be exposed rather than hard-coding.